nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/TestMSOfficeConverter.java

```diff
@@ -88,7 +87,8 @@ public void testAnyToTextExcelConverter() throws Exception {
 
     @Test
     public void testAnyToTextDocxConverter() throws Exception {
-        doTestAny2TextConverter("application/msword", "docx2text", "hello.docx");
+        doTestAny2TextConverter("application/vnd.openxmlformats-officedocument.wordprocessingml.document", "docx2text",
```

Comment:
Should we keep `application/msword` test?

Reply:
`application/msword` mime type is already tested.
Moreover a `.docx` file was never of type `application/msword`, it's for `.doc` file.